src/macaron/malware_analyzer/pypi_heuristics/heuristics.py

```diff
@@ -31,6 +31,9 @@ class Heuristics(str, Enum):
     #: Indicates that the setup.py file contains suspicious imports, such as base64 and requests.
     SUSPICIOUS_SETUP = "suspicious_setup"
 
+    #: Indicates that the package does not include a .whl file
+    WHEEL_PRESENCE = "wheel_presence"
```

Comment:
Would it be better to call this "WHEEL_ABSENCE"? All the others appear to be named based on the negative signal they represent when they fail, whereas this one passes when the wheel is present.

Reply:
That's fair, I can change this, the file-names, and the heuristic names to that since, for example `EmptyProjectLinkAnalyzer` passes when there is no empty project link.